concepticondata/conceptlists/Lionnet-1985-364.tsv

```diff
@@ -0,0 +1,365 @@
+Lionnet-1985-364-218	218	madre	mother	1216	MOTHER
+Lionnet-1985-364-219	219	madurarse, cocerse	ripen, become cooked	179	RIPEN
+Lionnet-1985-364-220	220	maíz	maize	506	MAIZE
+Lionnet-1985-364-221	221	mamar	suckle	1421	SUCK
+Lionnet-1985-364-222	222	mano	hand	1277	HAND
+Lionnet-1985-364-223	223	marido	husband	1200	HUSBAND
+Lionnet-1985-364-224	224	marrano	pig	1337	PIG
+Lionnet-1985-364-225	225	masticar	chew	321	CHEW
+Lionnet-1985-364-226	226	mata de fruta con cuernos	plant with horned fruits		
+Lionnet-1985-364-227	227	matar	kill	1417	KILL
+Lionnet-1985-364-228	228	matar (compl. plur.)	kill several		
+Lionnet-1985-364-229	229	metate	grindstone	2459	GRINDSTONE
+Lionnet-1985-364-230	230	mezcal	agave		
+Lionnet-1985-364-231	231	moco, catarro	mucus, slime, snot	208	NASAL MUCUS (SNOT)
+Lionnet-1985-364-232	232	mojarse	become wet	2324	GET WET
+Lionnet-1985-364-233	233	moler	grind	1033	GRIND
+Lionnet-1985-364-234	234	morder	bite	1403	BITE
+Lionnet-1985-364-235	235	morirse (suj. plur.)	die (several)		
+Lionnet-1985-364-236	236	morirse (suj sg.)	die (one)	1494	DIE
+Lionnet-1985-364-237	237	mosca	fly	1504	FLY (INSECT)
+Lionnet-1985-364-238	238	mucho	many	1198	MANY
+Lionnet-1985-364-239	239	murciélago	bat	1793	BAT
+Lionnet-1985-364-240	240	muslo	thigh	471	UPPER LEG (THIGH)
+Lionnet-1985-364-241	241	nariz	nose	1221	NOSE
+Lionnet-1985-364-242	242	nervio	nerve	2626	NERVE
+Lionnet-1985-364-243	243	nido	nest	539	NEST
+Lionnet-1985-364-244	244	nieve	snow	784	SNOW
+Lionnet-1985-364-245	245	noche, obscuridad	night, darkness	1233	NIGHT
+Lionnet-1985-364-246	246	nombre	name	1405	NAME
+Lionnet-1985-364-247	247	nube	cloud	1489	CLOUD
+Lionnet-1985-364-248	248	nuevo	new	1231	NEW
+Lionnet-1985-364-249	249	obscurecerse	darken		
+Lionnet-1985-364-250	250	ocote	pine		
+Lionnet-1985-364-251	251	ojo	eye	1248	EYE
+Lionnet-1985-364-252	252	oler (mal)	stink	1587	SMELL (STINK)
+Lionnet-1985-364-253	253	olote	degrained corncob		
+Lionnet-1985-364-254	254	ombligo	navel	1838	NAVEL
+Lionnet-1985-364-255	255	oreja	ear	1247	EAR
+Lionnet-1985-364-256	256	orinar	urinate	592	PISS
+Lionnet-1985-364-257	257	orines	urine	1142	URINE
+Lionnet-1985-364-258	258	oscuro	dark	706	DARK
+Lionnet-1985-364-259	259	pájaro	bird	937	BIRD
+Lionnet-1985-364-260	260	palma	palm	1183	PALM OF HAND
+Lionnet-1985-364-261	261	palo, madera	stick, wood	1295	STICK
+Lionnet-1985-364-262	262	panza	belly	1251	BELLY
+Lionnet-1985-364-263	263	papa	potato	593	POTATO
+Lionnet-1985-364-264	264	pararse	stand up	570	STAND UP
+Lionnet-1985-364-265	265	párpados	eyelid	17	EYELID
+Lionnet-1985-364-266	266	patear	kick	1135	KICK
+Lionnet-1985-364-267	267	pecho	chest	1592	CHEST
+Lionnet-1985-364-268	268	pedir	ask, request	1828	ASK (REQUEST)
+Lionnet-1985-364-269	269	peinarse	to comb	2070	COMB (VERB)
+Lionnet-1985-364-270	270	pelo, lana	hair, wool	2381	FEATHER OR FUR OR HAIR
+Lionnet-1985-364-271	271	pene	penis	1222	PENIS
+Lionnet-1985-364-272	272	perder en el juego	lose in a game	1660	LOSE
```

Comment:
Similar to WIN, I would unmap due to the specific game-related meaning of the original